py/tests/zip.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2018 The go-python Authors.  All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+doc="zip"
+a = [1, 2, 3, 4, 5]
+b = [10, 11, 12]
+c = [e for e in zip(a, b)]
+assert len(c) == 3
```

Comment:
Where has the check of the contents gone? :-)

Reply:
Oh, I've just understood that replace it by inline.. :)
I will revert the part :)

Reply:
@ncw Done!